### PR TITLE
drop rows with NULLs in forecast files

### PR DESCRIPTION
**Pull request by Serena-Wang**
- close issue #215

### CODE REVIEW

**Comment by elray1** [comment]
All the changes to the code seem plausible.  Can we add this case to a unit test?  (maybe 2 cases, one where no rows have NULL values and one with NULLs?)

File: tests/testthat/test-data/test-load_forecasts/data-processed/model-model1/2020-12-07-model-model1.csv [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+2020-12-07,4 wk ahead cum death,2021-01-02,69,quantile,0.975,2.0
+2020-12-07,4 wk ahead cum death,2021-01-02,69,quantile,0.99,3.0
+2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.01,
+2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.025,
+2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.05,
+2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.1,
+2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.15,
+2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.2,
+2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.25,
+2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.3,
```
Comment (elray1) [inline tests/testthat/test-data/test-load_forecasts/data-processed/model-model1/2020-12-07-model-model1.csv:101]:
```suggestion
2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.01,NULL
2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.025,NULL
2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.05,NULL
2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.1,NULL
2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.15,NULL
2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.2,NULL
2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.25,NULL
2020-12-07,1 wk ahead cum death,2020-12-12,60,quantile,0.3,NULL
```

Comment (elray1) [reply tests/testthat/test-data/test-load_forecasts/data-processed/model-model1/2020-12-07-model-model1.csv:101]: (The new format has text values of `NULL`)

**Review by elray1 — CHANGES_REQUESTED** [review]

**Review by elray1 — APPROVED** [review]
